tvb_multiscale/spiking_models/builders/factory.py

```diff
@@ -7,8 +7,8 @@
 
 from tvb_multiscale.config import CONFIGURED
 from tvb_multiscale.spiking_models.devices import DeviceSet
-from tvb_scripts.utils.data_structures_utils import flatten_tuple, ensure_list
-from tvb_scripts.utils.log_error_utils import initialize_logger, raise_value_error
+from tvb.simulator.plot.utils.data_structures_utils import flatten_tuple, ensure_list
```

Comment:
Also these, see more like utils for the `scripts` in general than utils for plotting. Right?